samples/sample-app-storage/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: "com.android.application"
 
 android {
     compileSdkVersion 29
-    buildToolsVersion "29.0.2"
+    buildToolsVersion "29.0.3"
```

Comment:
Is there any reason we are hard-coding this instead of using `buildToolsVersion "$androidBuildToolsVersion"` from root build.gradle file?